nemo/collections/nlp/models/machine_translation/mt_enc_dec_bottleneck_model.py

```diff
@@ -448,6 +448,8 @@ def eval_step(self, batch, batch_idx, mode, dataloader_idx=0):
             getattr(self, f'{mode}_loss_{dataloader_idx}')(
                 loss=eval_loss, num_measurements=num_measurements,
             )
+        inputs = [self.encoder_tokenizer.ids_to_text(src) for src in src_ids.detach().cpu().numpy()]
```

Comment:
Can we just use inputs returned by batch_translate (line 440), instead of recomputing detok input?

Reply:
Done.